test/runtests.jl

```diff
@@ -1,4 +1,36 @@
-using Test, Example
+using Test
+using DataFrames
+using SQLite
+using MySQL
+using SQLite
+using LibPQ
+using DBConnector
 
-@test hello("Julia") == "Hello, Julia"
-@test domath(2.0) ≈ 7.0
+@testset "_dbconnect function for SQLite" begin
+
+    conn= DBConnector._dbconnect(SQLite.DB, "../test/data/sqlite.db")
+    @test @isdefined conn
+    output = DBInterface.execute(conn, "SELECT age FROM PERSON WHERE name = 'John Doe';") |> DataFrame
+    out = output[1,1]
+    expected_output = 30;
+    @test out == expected_output
+
+end
+
+
+@testset "_dbconnect function for MySQL" begin
+    
+    conn = DBConnector._dbconnect(MySQL.Connection, ENV["MYSQL_HOST"], ENV["MYSQL_USER"], ENV["MYSQL_PASSWORD"], db="MySQL")
+
+    @test typeof(conn) == MySQL.Connection
+    @test isopen(conn)
+    close(conn)
+
+end
+
+@testset "_dbconnect function for LibPQ" begin
+
+    conn= DBConnector._dbconnect(LibPQ.Connection,  host = ENV["POSTGRES_HOST"], user = ENV["POSTGRES_USER"], dbname = "mimic", password = ENV["POSTGRES_PASSWORD"])
+    @test @isdefined conn
+
+end
```

Comment:
Could you make sure the tests work? I think the host string needs to be formatted again or something... Not entirely sure. What you can do for testing is try connecting to the database on the server from your local machine and establishing how the connection string should work.